### ISSUE TITLE
Improve the Accuracy of The Gender Taggger

### ISSUE THREAD

**Issue by djokester** [SangitaNLP/sangita#10]
The (word, gender) tuple is currently available [here](https://github.com/SangitaNLP/sangita/blob/master/sangita/hindi/corpora/gender.py)
In accordance with Issue #9 we will move this file to [Sangita Data](https://github.com/SangitaNLP/sangita_data) 
We will also create a new repository for Hindi Word Vectors and one for machine learning models. These will be referenced in a separate issue.  
Along with this we will remove the dependencies for Scikit Learn and work only with Keras. 
The task list is given below

- [ ] Move the gender.py to [Sangita Data](https://github.com/SangitaNLP/sangita_data) - Cakewalk. 

- [ ] Create a fresh set of word vectors and store it under a new repository especially for word vectors. - Pro.

- [ ] Train the word vectors against the gender tags, and store the model under a separate repository. - Intermediate. 

- [ ] Refactor the code [here](https://github.com/SangitaNLP/sangita/blob/master/sangita/hindi/gender.py), to accommodate these changes. - Intermediate.

**Comment by MansiBreja** [comment]
+1

**Comment by djokester** [comment]
For the word vectors we will use sentences from HDTB initially. 
Once @MansiBreja is done with [Issue](SangitaNLP/sangita_data#2) we can use the sentences from HindiMonoCorp too and train the word vectors again. 
Also we might need to scrape more sentences for this.
